data/clients.json

```diff
@@ -484,6 +484,12 @@
 			"acme_v2": "true",
 			"category": "Ruby"
 		},
+		{
+			"name": "Combine-LE: Generate and upload crt to CloudFlare(enterprise) and GCP.",
+			"url": "https://gitlab.com/6318613/letsencrypt",
```

Comment:
:wave: @michael-serchenia this ACME client name doesn't meet [the Let's Encrypt](https://letsencrypt.org/trademarks/) trademark requirements and can't be added to the client list under its current name.